ts/src/kucoin.ts

```diff
@@ -1227,14 +1228,13 @@ export default class kucoin extends Exchange {
 
```

Comment:
@ttodua options use `hf` but your methods uses `hfMigrated` I would rather use `hf` only

Reply:
right, done

Reply:
oh, static tests fail now, i'll handle that and ping you